### ISSUE TITLE
Are the APACHE scores available somewhere?

### ISSUE THREAD

**Issue by Franck-Dernoncourt** [mimic2/v3.0#56]
I wonder whether [APACHE scores](http://en.wikipedia.org/wiki/APACHE_II) are available somewhere, or whether someone has written some code to compute them.

There doesn't seem to be any used APACHE ITEMIDs in the table `CHARTEVENTS`:

```sql
 SELECT CHARTEVENTS.LABEL,  COUNT(*) cnt,  COUNT( DISTINCT( CHARTEVENTS.LABEL))
 FROM  MIMIC2V30.CHARTEVENTS
  WHERE LOWER(CHARTEVENTS.LABEL) LIKE '%apache%'
   GROUP BY CHARTEVENTS.LABEL
 ORDER BY cnt DESC ;
```

| "LABEL"                           | "CNT" | "COUNT(DISTINCT(CHARTEVENTS.LABEL))" | 
|-----------------------------------|-------|--------------------------------------| 
| "APACHEIII"                       | 12    | 1                                    | 
| "ApacheIV_LOS"                    | 12    | 1                                    | 
| "APACHE II Predecited Death Rate" | 10    | 1                                    | 
| "AgeApacheIIScore"                | 10    | 1                                    | 
| "APACHEII-Renal failure"          | 10    | 1                                    | 
| "OxygenApacheIIScore"             | 9     | 1                                    | 
| "MapApacheIIValue"                | 9     | 1                                    | 
| "SodiumApacheIIScore"             | 9     | 1                                    | 
| "TempApacheIIValue"               | 9     | 1                                    | 
| "MapApacheIIScore"                | 9     | 1                                    | 
| "HrApacheIIScore"                 | 9     | 1                                    | 
| "TempApacheIIScore"               | 9     | 1                                    | 
| "PotassiumApacheIIScore"          | 9     | 1                                    | 
| "HrApacheIIValue"                 | 9     | 1                                    | 
| "GcsApacheIIScore"                | 9     | 1                                    | 
| "PhApacheIIScore"                 | 9     | 1                                    | 
| "RRApacheIIValue"                 | 9     | 1                                    | 
| "HematocritApacheIIScore"         | 9     | 1                                    | 
| "AgeApacheIIValue"                | 9     | 1                                    | 
| "WbcApacheIIScore"                | 9     | 1                                    | 
| "CreatinineApacheIIScore"         | 9     | 1                                    | 
| "APACHE II"                       | 9     | 1                                    | 
| "RrApacheIIScore"                 | 9     | 1                                    | 
| "APACHE II PDR - Adjusted"        | 9     | 1                                    | 
| "DswfApacheScore"                 | 8     | 1                                    | 
| "HematocritApacheIIValue"         | 8     | 1                                    | 
| "SodiumApacheIIValue"             | 8     | 1                                    | 
| "PotassiumApacheIIValue"          | 8     | 1                                    | 
| "WBCApacheIIValue"                | 8     | 1                                    | 
| "ChpApacheIIScore"                | 8     | 1                                    | 
| "RRScore_ApacheIV"                | 7     | 1                                    | 
| "SodiumScore_ApacheIV"            | 7     | 1                                    | 
| "Creatinine_ApacheIV"             | 7     | 1                                    | 
| "RR_ApacheIV"                     | 7     | 1                                    | 
| "GlucoseScore_ApacheIV"           | 7     | 1                                    | 
| "MapScore_ApacheIV"               | 7     | 1                                    | 
| "TemperatureF_ApacheIV"           | 7     | 1                                    | 
| "WBCScore_ApacheIV"               | 7     | 1                                    | 
| "BunScore_ApacheIV"               | 7     | 1                                    | 
| "BUN_ApacheIV"                    | 7     | 1                                    | 
| "HR_ApacheIV"                     | 7     | 1                                    | 
| "Apache IV Age"                   | 7     | 1                                    | 
| "ApacheIV_Natural antilog"        | 7     | 1                                    | 
| "Sodium_ApacheIV"                 | 7     | 1                                    | 
| "Urine output_ApacheIV"           | 7     | 1                                    | 
| "Intubated_ApacheIV"              | 7     | 1                                    | 
| "GcsScore_ApacheIV"               | 7     | 1                                    | 
| "WBC_ApacheIV"                    | 7     | 1                                    | 
| "CreatinineApacheIIValue"         | 7     | 1                                    | 
| "MAP_ApacheIV"                    | 7     | 1                                    | 
| "OxygenScore_ApacheIV"            | 7     | 1                                    | 
| "BiliScore_ApacheIV"              | 7     | 1                                    | 
| "HtScore_ApacheIV"                | 7     | 1                                    | 
| "TempScore_ApacheIV"              | 7     | 1                                    | 
| "AgeScore_ApacheIV"               | 7     | 1                                    | 
| "ApacheIV_Mortality prediction"   | 7     | 1                                    | 
| "HCO3ApacheIIValue"               | 7     | 1                                    | 
| "AlbuminScore_ApacheIV"           | 7     | 1                                    | 
| "UrineScore_ApacheIV"             | 7     | 1                                    | 
| "PHPaCO2Score_ApacheIV"           | 7     | 1                                    | 
| "HrScore_ApacheIV"                | 7     | 1                                    | 
| "ChronicScore_ApacheIV"           | 7     | 1                                    | 
| "Glucose_ApacheIV"                | 7     | 1                                    | 
| "CreatScore_ApacheIV"             | 7     | 1                                    | 
| "Hematocrit_ApacheIV"             | 7     | 1                                    | 
| "FiO2ApacheIIValue"               | 6     | 1                                    | 
| "FiO2_ApacheIV"                   | 6     | 1                                    | 
| "PHApacheIIValue"                 | 6     | 1                                    | 
| "AaDO2ApacheIIValue"              | 6     | 1                                    | 
| "Apache IV PaFiRatio"             | 6     | 1                                    | 
| "Bilirubin_ApacheIV"              | 6     | 1                                    | 
| "PH_ApacheIV"                     | 5     | 1                                    | 
| "PCO2_ApacheIV"                   | 5     | 1                                    | 
| "Albumin_ApacheIV"                | 4     | 1                                    | 
| "Apache IV A-aDO2"                | 2     | 1                                    | 
| "PO2_ApacheIV"                    | 1     | 1                                    |

**Comment by mornin** [comment]
No. Apache scores are not calculated for the current release. Any users of MIMIC-II are welcome to contribute his code to calculate them or the calculated scores.

**Comment by alistairewj** [comment]
I have been working on code to calculate APACHE and other severity scores. Certain severity scores are not derivable (e.g. APACHE IV) because they require classifying patients into diagnostic categories. The code will be released after being updated to v3.0.

**Comment by Franck-Dernoncourt** [comment]
Moved to Stack Exchange: [Are the APACHE scores available somewhere?](http://opendata.stackexchange.com/q/6399/1652)
